xml/System.Diagnostics.Tracing/EventWrittenEventArgs.xml

```diff
@@ -66,20 +92,9 @@
         <ReturnType>System.Guid</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets the activity ID on the thread that the event was written to.</summary>
+        <summary>Gets the activity ID on the thread that the event was written to. See the Remarks section for more information. </summary>
```

Comment:
I think we don't usually add this given IntelliSense won't have remarks.

Reply:
Good point. I've removed them, @mairaw.